src/wrench/commands.py

```diff
@@ -56,12 +58,16 @@ def create_session_from_context(ctx_obj: Dict[str, Any]) -> GPGAuthSession:
     Return a `GPGAuthSession` from the given click context object.
     """
     session = GPGAuthSession(
-        gpg=ctx_obj['gpg'], server_url=ctx_obj['config']['auth']['server_url'], auth_uri='/auth/',
-        server_fingerprint=ctx_obj['config']['auth']['server_fingerprint'],
+        gpg=ctx_obj['gpg'], server_url=ctx_obj['config']['auth']['server_url']
     )
     session.auth = requests.auth.HTTPBasicAuth(
         ctx_obj['config']['auth']['http_username'], ctx_obj['config']['auth']['http_password']
     )
+    if session.server_fingerprint != ctx_obj['config']['auth']['server_fingerprint']:
+        logger.error('Server fingerprint don\'t match (%s != %s)',
+                     session.server_fingerprint,
+                     ctx_obj['config']['auth']['server_fingerprint'])
+        return False
```

Comment:
It would be better to raise an exception instead of returning False, catch it and display a meaningful error message to the user.